index.html

```diff
@@ -0,0 +1,132 @@
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <div class="normal-button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+
+      <div class="work-container">
+        <div class="card-work">
+          <div class="snapshoot"><img src="./images/Snapshoot_Portfolio(1).png" alt=""></div>
+          <div class="primary-text">
+            <h1>Multi-Post <br>Stories</h1>
+            <img src="./images/Frame2.png" alt="">
+            <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
+          </div>
+          <div>
+            <ul class="tags">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <div class="normal-button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+
+      <div class="work-container">
+        <div class="card-work">
+          <div class="snapshoot"><img src="./images/Snapshoot_Portfolio(2).png" alt=""></div>
+          <div class="primary-text">
+            <h1>Tonic</h1>
+            <img src="./images/Frame2.png" alt="">
+            <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
+          </div>
+          <div>
+            <ul class="tags">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <div class="normal-button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+
+      <div class="work-container">
+        <div class="card-work">
+          <div class="snapshoot"><img src="./images/Snapshoot_Portfolio(3).png" alt=""></div>
+          <div class="primary-text">
+            <h1>Multi-Post <br>Stories</h1>
+            <img src="./images/Frame2.png" alt="">
+            <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
+          </div>
+          <div>
+            <ul class="tags">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <div class="normal-button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-myself">
+      <div class="primary-text">
+        <h1>About <br>Myself</h1>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
+           of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="social">
+        <div class="let-connect"><a>LET'S CONNECT</a></div>
```

Comment:
- [ ] Kindly change "LET'S CONNECT" to a hx tag because it is a title. Keep coding 💪